lib/ovirt/service.rb

```diff
@@ -173,6 +173,7 @@ def probe_api_path(uri, path)
         request.get
       rescue RestClient::Exception => exception
         response = exception.response
+        return false if response.nil?
```

Comment:
@borod108 
yes, you are right, raising broke the contract. 

I just needed a protection against referencing the response when it is nil (which happens if the request times out). Returning false is enough in this case.